packages/deploy-config-sub-generator/src/utils/constants.ts

```diff
@@ -0,0 +1,3 @@
+// When deployment generator is bundled the namespacing is relative to the root generator
+export const bundledRootGeneratorName = '@sap/fiori:fiori-deployment';
```

Comment:
Should this be passed in from `@sap/generator-fiori`?

Reply:
Updated to use `opts.namespace`